UI/NpcList/NPCRow.gd

```diff
@@ -41,6 +35,22 @@ func getNpcID():
 		Log.error("Exception: NPCRow: character ID was not set")
 
 
+func getNpcName():
+	return _npcNameLabel.text
+
+
+func getNpcGender():
+	return _npcGenderLabel.text
+
+
+func getSubbyStatRaw() -> float:
+	return _npcSubbyStatRaw
+
+
+func getAmountOfChildren() -> int:
+	return int(_amountOfChildrenLabel.text)
```

Comment:
You can just create an int variable and store this so you don't have to cast strings to ints many times (and sorting something will do that -many- times x3)